federatedscope/llm/trainer/trainer.py

```diff
@@ -36,9 +63,21 @@ def _hook_on_batch_forward(self, ctx):
         ctx.loss_batch = CtxVar(loss, LIFECYCLE.BATCH)
         ctx.batch_size = CtxVar(len(labels), LIFECYCLE.BATCH)
```

Comment:
What should be the definition of value of `ctx.batch_size` when we using accelerator? In megetron-LM, there are micro-batch-size and global-batch-size, see similar discussion in [here](https://huggingface.co/docs/accelerate/concept_guides/performance#observed-batch-sizes)